api/google-search/pom.xml

```diff
@@ -44,6 +45,11 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springdoc</groupId>
+			<artifactId>springdoc-openapi-ui</artifactId>
+			<version>1.5.1</version>
+		</dependency>
```

Comment:
can we indent by 2 spaces through in vscode

Reply:
Dependencies before the xml are also in same indentation. I hope this is the issue. If anything else kindly elaborate else kindly resolve.